scripts/build.ts

```diff
@@ -52,8 +52,10 @@ class Builder {
   async buildRenderer() {
     process.env.VUE_CLI_SERVICE_CONFIG_PATH = 'pkg/rancher-desktop/vue.config.js';
     await simpleSpawn(
-      'node_modules/.bin/vue-cli-service',
+      process.execPath,
       [
+        '--stack-size=16384',
```

Comment:
Would maybe still be nice if this setting could be overridden via env variable or something.

Also the same setting is applied in multiple place; should they share a constant definition?

Reply:
That would be good, but it seemed like we wanted a quick solution to unblock your PR while we figured things out. I'll do it as a follow-up?